frontend/lib/widgets/match_alert.dart

```diff
@@ -128,6 +131,21 @@ class MatchAlert extends StatelessWidget {
                 ],
               ),
             ),
+            SizedBox(
+              width: size.width * .8,
+              child: Expanded(
+                child: TextButton(
```

Comment:
The button doesn't respect the UI, I can see the text but not the button

Reply:
Should be fixed now